ganttproject/src/biz/ganttproject/impex/csv/XlsWriterImpl.java

```diff
@@ -7,60 +26,59 @@
 import org.apache.poi.ss.usermodel.Sheet;
 import org.apache.poi.ss.usermodel.Workbook;
 
-import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.OutputStream;
 
 /**
  * @author akurutin on 04.04.2017.
  */
-public class XlsWriterImpl implements XlsWriter {
-    private Workbook workbook;
-    private Sheet sheet;
-    private final OutputStream stream;
+public class XlsWriterImpl implements SpreadsheetWriter {
+  private Workbook myWorkbook;
+  private Sheet mySheet;
+  private final OutputStream myStream;
 
 
-    private Row currentRow = null;
-    private int nextRowInd = 0;
-    private int nextCellInd = 0;
+  private Row myCurrentRow = null;
+  private int myNextRowInd = 0;
+  private int myNextCellInd = 0;
 
 
-    public XlsWriterImpl(OutputStream stream, CSVFormat format) {
-        this.stream = stream;
-        workbook = new HSSFWorkbook();
-        sheet = workbook.createSheet();
-    }
+  public XlsWriterImpl(OutputStream myStream, CSVFormat format) {
```

Comment:
... and function arguments should _not_ have prefix `my`.

The purpose of this prefix is to be able to distinguish instance variable from local variable or function argument by merely reading code without IDE assistance, e.g. in the code review interface